security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -10,11 +10,66 @@ The {cert-manager-operator} is a cluster-wide service that provides application
 
 These release notes track the development of {cert-manager-operator}.
 
-:FeatureName: The {cert-manager-operator}
-include::snippets/technology-preview.adoc[leveloffset=+1]
-
 For more information, see xref:../../security/cert_manager_operator/index.adoc#cert-manager-operator-about[About the {cert-manager-operator}].
 
+[id="cert-manager-operator-release-notes-1.10.2"]
+== Release notes for {cert-manager-operator} 1.10.2
+
+// TODO: Confirm date closer to GA
+Issued: 2023-03-23
+
+The following advisory is available for the {cert-manager-operator} 1.10.2:
+
+// TODO: Update
+* link:https://access.redhat.com/errata/RHEA-2023:TODO[RHEA-2023:TODO]
+// https://errata.devel.redhat.com/advisory/109104
+
+For more information, see the link:https://cert-manager.io/docs/release-notes/release-notes-1.10[cert-manager project release notes for v1.10].
+
+[id="cert-manager-operator-1.10.2-new-features-and-enhancements"]
+=== New features and enhancements
+
+This is the general availability (GA) release of the {cert-manager-operator}.
+
+* The following issuer types are supported:
+** ACME
+** CA
+** Self-signed
+
+* The following ACME challenge types are supported:
+** DNS-01
+** HTTP-01
+
+* The following DNS-01 providers for ACME issuers are supported:
+** Amazon Route 53
+** Azure DNS
+** Google Cloud DNS
+
+// TODO: Any other features to list or things to call out as supported? proxy support?
+
+[id="cert-manager-operator-1.10.2-bug-fixes"]
+=== Bug fixes
+
+* Previously, the `unsupportedConfigOverrides` field replaced user-provided arguments instead of appending them. Now, the `unsupportedConfigOverrides` field properly appends user-provided arguments. (*link:https://issues.redhat.com/browse/CM-23[CM-23]*)
++
+[WARNING]
+====
+Using the `unsupportedConfigOverrides` section to modify the configuration of an Operator is unsupported and might block cluster upgrades.
+====
+
+* Previously, the {cert-manager-operator} did not inject the necessary cluster wide egress proxy configuration. With this release, the {cert-manager-operator} now properly supports injecting custom CA certificates and detects cluster wide egress proxy configuration. (*link:https://issues.redhat.com/browse/CM-37[CM-37]*)
+
+* Previously, the {cert-manager-operator} was installed as a cluster Operator. With this release, the {cert-manager-operator} is now properly installed as an OLM Operator. (*link:https://issues.redhat.com/browse/CM-35[CM-35]*)
+
+[id="cert-manager-operator-1.10.2-known-issues"]
+=== Known issues
+
+* Using `Route` objects is not fully supported. Currently, {cert-manager-operator} integrates with `Route` objects by creating `Ingress` objects through the Ingress Controller. (link:https://issues.redhat.com/projects/CM/issues/CM-16[*CM-16*])
```

Comment:
https://issues.redhat.com/projects/CM/issues/CM-16 will display other issues on the left. https://issues.redhat.com/browse/CM-16 is consistent with previous paragraphs' links.
```suggestion
* Using `Route` objects is not fully supported. Currently, {cert-manager-operator} integrates with `Route` objects by creating `Ingress` objects through the Ingress Controller. (link:https://issues.redhat.com/browse/CM-16[*CM-16*])
```